platforms/nuttx/src/px4/stm/stm32h7/include/px4_arch/micro_hal.h

```diff
@@ -65,5 +65,6 @@ int stm32h7_flash_lock(size_t addr);
 int stm32h7_flash_unlock(size_t addr);
 int stm32h7_flash_writeprotect(size_t block, bool enabled);
 #define  stm32_flash_lock() stm32h7_flash_lock(PX4_FLASH_BASE)
+#define APX4_ADC_INTERNAL_TEMP_SENSOR_CHANNEL 17 //Valid for ADC3 on H7x3
```

Comment:
Once this is corrected and the code to support it is added this should come in.

@dagar said add the temperature to `top` for the next PR. From there as a subcategory of system integrity like we discussed today.

Reply:
I was talking about either `ORB_ID(cpuload)` or a new standalone topic (if there's more relevant data).

https://github.com/PX4/PX4-Autopilot/blob/3ad0da51c4841fb20ced4d8e205d9abb5541e889/src/modules/load_mon/LoadMon.cpp#L220-L229